docs/CHANGELOG.asciidoc

```diff
@@ -19,3 +19,255 @@ This section summarizes the changes in each release.
 == {kib} 8.0.0-alpha1
 
 coming[8.0.0]
+
+The following changes have been released for the first time in {kib} 8.0.0-alpha1.
+
+[float]
+[[breaking-changes-8.0.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and performance. Before you upgrade to 8.0.0, review the breaking changes, then mitigate the impact to your application.
```

Comment:
Not sure if this just standard verbage or not, but it might be worth directing users to the Upgrade Assistant here?

Reply:
@joshdover this breaking changes section reflects what we have in the [Elasticsearch Migration Guide](https://www.elastic.co/guide/en/elasticsearch/reference/master/migrating-8.0.html#breaking-changes-8.0). This section is more for reviewing the breaking changes, then updating whatever you need to make sure your upgrade is successful instead of upgrading. We keep them separate in the Kibana docs so users know they don't have mitigate all breaking changes if they aren't necessary. We keep all required upgrade content in the Upgrade Assistant section.